spec/factories/user.rb

```diff
@@ -0,0 +1,11 @@
+require 'factory_girl'
+
+FactoryGirl.define do
+  factory :user do
+    email "simon@example.com"
+    name  "Simon"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.